lib/matplotlib/colors.py

```diff
@@ -742,6 +742,107 @@ def func_r(x):
 
         return LinearSegmentedColormap(name, data_r, self.N, self._gamma)
 
+    def join(self, other, name=None, frac_self=None, N=None):
```

Comment:
I had this idea as well, but thought reviewers would find it 'too cute'! It's included now.